lib/chartmogul/v1/customers.rb

```diff
@@ -60,6 +64,30 @@ def update(customer_id, attributes = [])
         end
       end
 
+      class EmailAttributes < Customers
+        # Public: Create Custom Attributes for a Customer
+        #
+        # See: https://dev.chartmogul.com/docs/create-custom-attributes-for-customer
```

Comment:
Misprint, should be `https://dev.chartmogul.com/docs/create-custom-attributes-for-customer-with-email`